commands/audit/sca/python/python.go

```diff
@@ -281,6 +306,18 @@ func executeCommand(executable string, args ...string) error {
 	return nil
 }
 
+func executeCommandWithOutput(executable string, args ...string) (string, error) {
+	installCmd := exec.Command(executable, args...)
+	maskedCmdString := coreutils.GetMaskedCommandString(installCmd)
+	log.Debug("Running", maskedCmdString)
+	output, err := installCmd.CombinedOutput()
+	if err != nil {
+		sca.LogExecutableVersion(executable)
+		return string(output), errorutils.CheckErrorf("%q command failed: %s - %s", maskedCmdString, err.Error(), output)
+	}
+	return string(output), nil
+}
```

Comment:
```suggestion
func executeCommand(executable string, args ...string) (string, error) {
	installCmd := exec.Command(executable, args...)
	maskedCmdString := coreutils.GetMaskedCommandString(installCmd)
	log.Debug("Running", maskedCmdString)
	output, err := installCmd.CombinedOutput()
	if err != nil {
		sca.LogExecutableVersion(executable)
		return string(output), errorutils.CheckErrorf("%q command failed: %s - %s", maskedCmdString, err.Error(), output)
	}
	return string(output), nil
}
```

Remove code duplication, use already existing func, just ignore output content when not needed